### PR TITLE
Enable translations for html templates

### PR DESCRIPTION
**Pull request by joncasdam**
Using jinja feature, check language and enable translation for incomplete message and menu in the templates.
English remain as the fallback/standard language.

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -42,33 +42,11 @@
...
-              <a class="nav-link" href="/contributing/">Contributing{% if this.is_child_of('/contributing') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-            <li class="nav-item{% if this.is_child_of('/contact') %} active{% endif %}">
-              <a class="nav-link" href="/contact/">Contact{% if this.is_child_of('/contact') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-          </ul>
-        </div>
-      </div>
-    </nav>
+    {% if this.alt == 'pt' %}
```
Comment (freakboy3742) [inline templates/layout.html]: Is there any way to make this automated, rather than requiring a manual rollout for every translation language?

Comment (joncasdam) [inline templates/layout.html]: Of course it makes total sense. I'll update.

**Comment by joncasdam** [comment]
changes amended, @freakboy3742

**Comment by joncasdam** [comment]
Added commit with fix for https://github.com/pybee/pybee.github.io/issues/102

**Comment by joncasdam** [comment]
@glasnt or @freakboy3742 did you had a chance to look at this?

**Comment by goanpeca** [comment]
@joncasdam I am helping with the Spanish translation and I was wondering if is this PR is ready?

**Comment by joncasdam** [comment]
yes, it is ready, @goanpeca

**Comment by goanpeca** [comment]
Any further comments @freakboy3742 ?

**Comment by goanpeca** [comment]
@joncasdam actually there are many locations in templates where there is hardcoded text, maybe we can use this PR to create bags for those?

Some examples
### 404.html
```
<h1>404: Page not found</h1>
```
### community.html
```
    <h2>Commercial options</h2>
    <p>Although BeeWare is an open source project, there are companies and individuals who can provide paid support, training, and consulting.</p>
```

**Comment by joncasdam** [comment]
I was preparing another PR with more fixes and adding more bags. For this I'd like to enable language selection. What do you think?

**Comment by goanpeca** [comment]
@joncasdam sure, whatever gets us going :-)

I am decoupling the home template. I started https://github.com/pybee/pybee.github.io/pull/158

Plan to add flowblocks for the home content so we can translate it. Also creating a bag per template for words/headers that are missing translations. Is having a bag per template/per macro something you agree on? Might lead to duplication, but while there are not so many templates its more organized. Thoughts?

**Comment by joncasdam** [comment]
I can open a pr with my diff for pt-br then, if applicable, you could see it's worthy to base on it to use bags or I can, without problem, rebase mine later. 
Since in the coming 10 days (at least) I wont have much chance to do something, I'd go for the later. I rebase mine later based on your work - if that's ok.

**Comment by goanpeca** [comment]
@joncasdam I think databags are a good idea, I will just open a new PR with other databags for other pages, lets just hope @freakboy3742 can merge this one soon :-)

**Pull request merged**